README.md

```diff
@@ -8,12 +8,27 @@ to be added to Drupal core and installed using [Composer](https://getcomposer.or
 below use the [Apigee Devportal Kickstart composer project](https://github.com/apigee/devportal-kickstart-project-composer)
 to install Drupal core and this installation profile to create a site.
 
-# Prerequisites
+## Module status
+
+The core functionality of this module is in active development. The features and functionality will be changing as we
+continue work.
+
+We encourage you to download and evaluate the module, and to use our 
+[GitHub issue queue](https://github.com/apigee/apigee-devportal-kickstart-drupal/issues) to give feedback, ask questions, 
+or log issues.
+
+## Planned features
+* **Demonstration theme & content:** Demonstration content with the look and feel of a branded developer portal
+* **API Documentation:** Example of API documentation published on site using OpenAPI specifications
+* **Apigee Edge configuration:** The ability to configure the site to an Apigee organization during installation
+* **Blog and Forums:** Demonstration of a blog and forum on the developer portal to communicate with developers
```

Comment:
Are blog+forums not ootb? Not sure what the feature is beyond that.

Reply:
The blog and forums modules exist, but we want to add and configure it to show how it can look. For example, adding forum "recent" topics to home page